compiler/rustc_session/src/options.rs

```diff
@@ -1188,6 +1188,8 @@ options! {
         "compile without linking"),
     no_parallel_llvm: bool = (false, parse_no_flag, [UNTRACKED],
         "run LLVM in non-parallel mode (while keeping codegen-units and ThinLTO)"),
+    no_unique_section_names: bool = (false, parse_bool, [TRACKED],
```

Comment:
I didn't use `parse_no_flag` - unlike the other `no_*` flags above - because the `parse_no_flag` function says its use is deprecated, so I made it a plain bool. Also, not entirely sure what tracking is for, so I set it to `[TRACKED]` because `function_sections` is also tracked.